index.html

```diff
@@ -49,180 +49,226 @@ <h2 class="hex connect">LET’S CONNECT</h2>
-                <div class="see-project">
+            </ul>
+            <div class="see-project">
                 <button type="button" class="hex">Get my resume</button>
-                </div>
-                <div class="languages-list">
+            </div>
+            <div class="languages-list">
                 <ul>
-                <li class="lang-title">
-                <p class="langusge">Languages</p>
-                <img class="allow-down" src="images/Icons/vector.png" alt="allow-down" />
-                </li>
-                <li class="lang">
-                <img src="images/langusges/js.png" alt="js" />
-                <p>JavaSript</p>
-                </li>
-                <li class="lang">
-                <img src="images/langusges/html.png" alt="js" />
-                <p>HTML</p>
-                </li>
-                <li class="lang">
-                <img src="images/langusges/css.png" alt="js" />
-                <p>CSS</p>
-                </li>
-                <li class="frameworks">
-                <p class="langusge">Frameworks</p>
-                <p class="sign">></p>
-                </li>
-                
-                <li class="frameworks">
-                <p class="langusge">Skills</p>
-                <p class="sign">></p>
-                </li>
+                    <li class="lang-title">
+                        <p class="langusge">Languages</p>
+                        <img class="allow-down" src="images/Icons/vector.png" alt="allow-down" />
+                    </li>
+                    <li class="lang">
+                        <img src="images/langusges/js.png" alt="js" />
+                        <p>JavaSript</p>
+                    </li>
+                    <li class="lang">
+                        <img src="images/langusges/html.png" alt="js" />
+                        <p>HTML</p>
+                    </li>
+                    <li class="lang">
+                        <img src="images/langusges/css.png" alt="js" />
+                        <p>CSS</p>
+                    </li>
+                    <li class="frameworks">
+                        <p class="langusge">Frameworks</p>
+                        <p class="sign"></p>
+                    </li>
+                    <li class="frameworks">
+                        <p class="langusge">Skills</p>
+                        <p class="sign"></p>
+                    </li>
                 </ul>
-                </div>
             </div>
-
+        <section class="contact-me"> 
+            <div class="contact-background">
+                <h2>Contact me</h2>
+                <p>
+                    If you have an application you are interested in developing, a feature that you need to build or a project that needs coding. I'd love to help with
+                </p>           
+                <form>
+                    <input type="text" id="fname" name="fname" placeholder="lorem Ipsum" maxlength="30" required><br>
```

Comment:
- [ ] Dear @Jonathan-kiprono, For this exercise then form is expected to use the action and method attribute to integrate with [formspree service](https://formspree.io/html). Please make sure to set,  `action` to `https://formspree.io/f/{form_id}` and  `method` to `POST`. 
- [ ] Please, once yo set the above action check that your form is getting submitted with the right values.